src/content/docs/telemetry-data-platform/manage-data/manage-data-retention.mdx

```diff
@@ -366,9 +341,54 @@ The following table shows the default retention settings for our [New Relic One
   </tbody>
 </table>
 
+## Adjust retention values [#adjust-retention]
+
+You can adjust your retention periods to better support your data needs and meet compliance requirements. You might reduce retention values to optimize query performance. Or, you might increase retention for some data types to support more granular analysis over longer periods of time.  
+
+The ability to edit data retention periods requires a specific [edition of New Relic](https://newrelic.com/pricing).
+
+* Legacy accounts must have Insights Pro. 
+
+* New Relic One accounts have these edition considerations:
+
+  * Standard edition: your data retention is fixed, and falls between 8 and 395 days (about 13 months) of retention, depending on which capabilities of the New Relic platform you’re using. The prescribed retention levels ensure that you can explore your data and experience the value of New Relic. If you need to raise some of the retention limits for your particular business needs, you can purchase additional retention.
+  * Pro or Enterprise: If after you finalize your contracted retention you realize you need to make changes, you can adjust your telemetry data retention settings yourself.
+
+### User roles and retention
+
+In order to edit retention, a V1 user must be the **Owner** or have the **Data Retention Manager** add-on role. A V2 user must have the **billing_user** role. 
```

Comment:
is this capitalization vs. lower case intentional? I'm not sure how it all looks anymore, sorry!